torch/nn/utils/rnn.py

```diff
@@ -30,14 +30,18 @@ class PackedSequence(PackedSequence_):
             information about the batch size at each sequence step
 
     """
-    def __new__(cls, *args):
+    def __new__(cls, data, batch_sizes=None):
+        # Using an `*args` and an if statement on `len(args)` breaks BC of the
+        # calling pattern `PackedSequence(data=..., batch_sizes=...)`, so we
+        # have to provide two arguments with names `data` and `batch_sizes`.
+        #
```

Comment:
I wouldn't mention using `*args` anywhere in here. It won't be there once we merge this, and so the comment only ends up being confusing. Just say that we should try to preserve BC here, and so you should think twice before you change arg names (even though the code below ends up being weird this way).